lib/client/session.go

```diff
@@ -149,7 +149,8 @@ func newSession(client *NodeClient,
 		defer ns.closeWait.Done()
 
 		<-ns.closer.C
-		if isFIPS() {
+
+		if isFIPS() || isAuthServerBoring(client) {
```

Comment:
I don't necessarily have a better suggestion to offer, but adding a network call on exit makes me mildly uneasy. I could see it leading to hanging clients and screens that don't reliably clear after you suspend your laptop, for example.

Maybe we could call it at the beginning of the session and cache the value?